src/app/page.tsx

```diff
@@ -43,40 +46,20 @@ export default function Home() {
             </Link>
           </li>
           <li>
-            <a
-              href="https://www.github.com/phantomstudios/css-components"
-              rel="external noreferrer"
-              target="_blank"
-            >
+            <a href={githubLink} rel="external noreferrer" target="_blank">
               <GoMarkGithub />
               GitHub
             </a>
           </li>
         </ul>
         <CodeClipboard text="npm i @phntms/css-components" />
+        <Version>
+          Currently <strong>v{version}</strong>
+        </Version>
       </Hero>
-      <CodeExample />
-      <HeroFeatures />
-      <Stats
-        stats={[
-          {
-            title: "Bundle Size",
-            value: "2.3kb",
-          },
-          {
-            title: "Bundle Size (GZIP)",
-            value: "961b",
-          },
-          {
-            title: "Runtime interpolations",
-            value: "Zero",
-          },
-          {
-            title: "Variants",
-            value: "∞",
-          },
-        ]}
-      />
+      <CodeExample {...codeExample} />
```

Comment:
maybe these three components should import their content in the components? not sure we need to import in the page and pass as a prop

Reply:
Cool got you